src/redux/feature/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,53 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import { getRequest } from '../../../services/apiService';
+
+export const fetchRockets = createAsyncThunk(
+  'rockets/fetchRockets',
+  async () => {
+    const response = await getRequest('v3/rockets');
+    return response || {};
+  },
+);
+
+const initialState = {
+  rockets: [],
+  loading: false,
+  rocketsInProcess: null,
```

Comment:
rocketsInProcess is not used in the application so kindly remove all unnecessary codes as it would make your code hard to read and Maintain.